www/templates/main-goals.html

```diff
@@ -54,6 +55,7 @@
               <span ng-show="profile.items.currentPet" class="current-pet Pet-{{profile.items.currentPet}}"></span>
             </div>
           </figure>
+          <!--Habitica jade to html code ends here-->
```

Comment:
Also, can you add a short README  that describes the steps to export the habitica jade to HTML?
- load the page from the server
- click on the avatar
- .....